### ISSUE TITLE
Sidebar height is broken on some pages

### ISSUE THREAD

**Issue by tomimarkus991** [TPT-Loane/TPT-Loane#66]
Sidebar height on login page:
![image](https://user-images.githubusercontent.com/48998818/116810973-64fcfe80-ab4f-11eb-8aa3-f265e64d7020.png)

Also when scrolling down the page Sidebar also scrolls down. this is caused by Padding-Bottom in the default layout component.
![image](https://user-images.githubusercontent.com/48998818/116811034-c45b0e80-ab4f-11eb-897d-c70e4e25f366.png)

**Comment by GeorgDV** [comment]
The height issue has a fix incoming from [here](https://github.com/TPT-Loane/TPT-Loane/pull/56/files#diff-2ac13efbc59b060e78f8e6b7f1df09c386e7b06e8e3f1d1920abbce6740654fcR18). Although it's not perfect. It would be prettier if the length of the `Flex` container would be full height of available page height - `100%` instead of `100vh`. But it doesn't work as simply.

**Comment by Kahuka** [comment]
@GeorgDV Has this been fixed?

**Comment by tomimarkus991** [comment]
> 
> 
> @GeorgDV Has this been fixed?

I believe the first one is. The second one is not

**Comment by GeorgDV** [comment]
> > @GeorgDV Has this been fixed?
> 
> I believe the first one is. The second one is not

Yep, second one is still there, but overall its an annoying css issue.
